lib/gui/components/progress-button/templates/progress-button.tpl.html

```diff
@@ -3,5 +3,5 @@
     'progress-button--striped': striped && striped != 'false'
   }">
     <span class="progress-button__content" ng-transclude></span>
-    <span class="progress-button__bar" ng-style="{ width: Math.min(100, percentage) + '%' }"></span>
+    <span class="progress-button__bar" ng-style="{ width: progress.min(100, percentage) + '%' }"></span>
```

Comment:
I think it might need to be enclosed in parenthesis, otherwise `%` will be added to the else condition.